class-validator/src/__tests__/class-validator.ts

```diff
@@ -145,3 +159,30 @@ it('validate data with validator option', async () => {
 
   expect(result).toMatchSnapshot();
 });
+
+it('should return from classValidatorResolver with `excludeExtraneousValues` set to true', async () => {
+  class SchemaTest {
+    @Expose()
+    @IsNumber({}, { message: `Must be a number`, always: true })
+    random: number;
+  }
+
+  const result = await classValidatorResolver(SchemaTest, {
+    transformer: {
+      excludeExtraneousValues: true,
```

Comment:
I don't find any documentation about `excludeExtraneousValues` and `extraneousField`. Do you have a link that explains these options?